### ISSUE TITLE
Motion Embedding demo

### ISSUE THREAD

**Issue by VictoriYang** [nv-tlabs/vid2player3d#3]
Thank you for your wonderful job! 

Could you please also provide the data preparation code and demo data used for Motion Embedding Phase.  Thank you!

**Comment by Haotianz94** [comment]
Unfortunately the tennis motion data is still not available due to license issues and we do not foresee the chance of releasing that data in the near future.

As for the data preparation code, we do not plan to release it since it depends on the output of the preprocessing stage, which makes it hard to follow. The best way to understand the data format is through the [__init__](https://github.com/nv-tlabs/vid2player3d/blob/a8b815fa4474f4b672b6e437b55cbe2aeb9001f3/vid2player/motion_vae/dataset.py#L15) function of the dataset loader. Basically we store different pose fields (positions, angles, etc) as sparate numpy arrays and store the index of the first frame of each motion sequence in a json file.

**Comment by VictoriYang** [comment]
Thanks a lot!
